Gemfile

```diff
@@ -24,6 +25,7 @@ group :development, :test do
   gem 'pry'
   gem 'simplecov'
   gem 'shoulda-matchers'
+  gem 'orderly'
```

Comment:
Added for testing order in items index spec. Didn't ask permission. Seems pretty standard 🤷‍♀️

Reply:
ask for forgiveness later

Reply:
I asked last project, they approved